torch/csrc/jit/script/init.cpp

```diff
@@ -501,12 +501,19 @@ void initJitScriptBindings(PyObject* module) {
     .def("params", &Method::params)
     .def("graph_for", [](Method& self, py::args args) {
       return self.graph_for(createVariableTensorList(args));
-    });
+    })
+    .def("pretty_print_schema", &Method::prettyPrintSchema);
 
-  m.def("_jit_script_compile", [](Def def, ResolutionCallback rcb) {
-    return compileFunction(def, pythonResolver(rcb));
+  m.def("_jit_script_compile_pure_fn", [](Module &m, DefAndTypes def, ResolutionCallback rcb) {
```

Comment:
"pure fn" is perhaps not the best name, because it can have side effects. Just calling it *function* might be enough (as opposed to a *method* on a module).